src/plugins/data/public/types.ts

```diff
@@ -57,20 +60,61 @@ export interface DataPublicPluginSetup {
   __enhance: (enhancements: DataPublicPluginEnhancements) => void;
 }
 
+/**
+ * data plugin prewired UI components
```

Comment:
nit, for consistency with other comments.

```suggestion
 * Data plugin prewired UI components
```